### ISSUE TITLE
[Bug]: Tests in "Readme" not working

### ISSUE THREAD

**Issue by Heiko-H** [pimcore/skeleton#154]
### Expected behavior

As written in the readme file of Pimcore 11.0.0: 
`docker compose run --rm test-php vendor/bin/codecept run -vv`

```
Running installation. You can find a detailed install log in var/log/test.log

Marking all migrations as done ...

 11/11 [============================] 100%

```

### Actual behavior

```
In Kernel.php line 594:
  Unable to create the "cache" directory (/var/www/html/var/installer/cache).

```

There is a permission problem.

### Steps to reproduce

Just follow the installation steps of the readme file and on step 5: 

`docker compose run --rm test-php vendor/bin/codecept run -vv`

**Comment by jdreesen** [comment]
Did you configure the `user` in your `docker-compose.yaml` as stated in the readme?

**Comment by Heiko-H** [comment]
We found a solution so that the tests will be successful on the local machine: 

Add the following line "user" into the docker-compose.yaml: 
```
test-php:
        user: '0:0'
        profiles: [ 'test' ]
```

So the container test-php will be executed as root. 
Is there a PR necessary for the docker-compose.yaml or is it a bug of the "test-php" container?

**Comment by Heiko-H** [comment]
> Did you configure the `user` in your `docker-compose.yaml` as stated in the readme?

Yes i changed the user to 1000:1000 (my uid and gid) in the containers "php" and "db".
Normal installation works fine but the codeception tests fail.

**Comment by yariksheptykin** [comment]
thanks for reporting the issue @Heiko-H . I am able to reproduce it locally. It looks like this is caused by named volumes mounted into `php-test` container on `var/` and `public/var/`. When docker engine creates those volumes it assigns them to the root user.

The solution with adding `user: '0:0'` or `user: root` to the `docker-compose.yaml` seem fine to me. This will lead to tests running as root in the container. If the tests or the installer create any files outside of `var/` or `public/var`, those will be owned by root, which might cause some inconvenience on the host system.

Alternatively we can update the README by adding a command to correct filesystem permissions of those docker volumes:

```
5. Run codeception tests:
   * `docker compose run --user=root --rm test-php chown -R $(id -u):$(id -g) var/ public/var/`
   * `docker compose run --rm test-php vendor/bin/pimcore-install -n`
   * `docker compose run --rm test-php vendor/bin/codecept run -vv`
```

Either way, this is a bug in the test setup instructions and we should fix it in a PR. I suggest we go with the "README option".

**Comment by yariksheptykin** [comment]
The issue is reproducible in CI as well https://github.com/pimcore/skeleton/actions/runs/5657657325/job/15327222075#step:7:31

**Comment by dvesh3** [comment]
Fixed by #162
